### DISCUSSION TITLE
Uploading images

### DISCUSSION THREAD

**Discussion by SharonD19** [TheAliceProject/alice3#253] [General]
I would like to add image to the Alice program.  How do I download images to the program.

**Comment by dabeshouse** [comment]
The Billboard model in the Shapes/Text tab in the gallery allows images to be imported onto the front and a second image on the back.
The other shapes in that gallery tab also support images being applied to them.

**Comment by djslater107** [comment]
In Alice 2, go to the File menu and select the Import… option. You will then navigate through your file system to find the image you wish to import… See this video for Alice 2: https://www.youtube.com/watch?v=cO-b24t4khw

For Alice 3, go to the Gallery in the Scene Editor, and select the Shapes / Text tab. In that area elect new Billboard. Clicking on the Front tab menu will give you an Import Image option.

See this video for Alice 3: https://www.youtube.com/watch?v=XeO2GOqbonM
